e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/PublishSingleNodesTestTheory.cs

```diff
@@ -35,6 +35,11 @@ public class PublishSingleNodesTestTheory
         public async Task Test_CollectOAuthToken() {
             var token = await TestHelper.GetTokenAsync(_context);
             Assert.NotEmpty(token);
+
+            // We will wait for microservices of IIoT platform to be healthy.
+            var millisecondsDelay = 600 * 1000;
+            var cts = new CancellationTokenSource(millisecondsDelay);
+            await TestHelper.WaitForServicesAsync(_context.IIoTPlatformConfigHubConfig, cts.Token);
```

Comment:
Please move this code to first test that is working with IIoT PLatform `Test_RegisterOPCServer_Expect_Success`

Reply:
Done.